minikube/minikube.bzl

```diff
@@ -0,0 +1,66 @@
+def _minikube_tool_impl(ctx):
+    os_build_name = "linux"
+    if ctx.os.name.startswith("mac"):
+        os_build_name = "darwin"
+    ctx.download(
+        "https://storage.googleapis.com/minikube/releases/latest/minikube-%s-amd64" % (os_build_name),
+        output='minikube',
+        executable=True,
+    )
+    ctx.file("BUILD", 'exports_files(["minikube"])')
+
+minikube_tool = repository_rule(
+    implementation = _minikube_tool_impl,
+)
+
+def _minikube_start_impl(ctx):
+    script = "{} -p airy-core start --driver=docker --cpus=4 --memory=7168 --container-runtime=containerd --ports=80:80 --extra-config=apiserver.service-nodeport-range=1-65535".format(ctx.executable._minikube_binary.path)
```

Comment:
Probably all these stuff should be submitted as attributes, particularly the name of the profile.